libtock/usb.c

```diff
@@ -1,15 +1,36 @@
 #include "usb.h"
 
 int usb_exists(void) {
-  return command(DRIVER_NUM_USB, 0, 0, 0) >= 0;
+  syscall_return_t com = command2(DRIVER_NUM_USB, 0, 0, 0);
+  if (com.type == TOCK_SYSCALL_SUCCESS) {
+    return TOCK_SUCCESS;
+  } else if (com.type > TOCK_SYSCALL_SUCCESS) {
+    // Returned an incorrect success code
+    return TOCK_FAIL;
+  } else {
+    return tock_error_to_rcode(com.data[0]);
+  }
```

Comment:
before this returned 1 if the driver was present and 0 otherwise, now it returns 0 if the driver is present

Reply:
Fixed.